src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-dashboard.component.html

```diff
@@ -5,12 +5,26 @@ <h4>
     <jhi-assessment-filters (filterChange)="updateFilteredSubmissions($event)" [submissions]="submissions"></jhi-assessment-filters>
     <jhi-assessment-warning [exercise]="exercise" [submissions]="filteredSubmissions"></jhi-assessment-warning>
     <h4>
-        <small [ngStyle]="assessedSubmissions === submissions.length && submissions.length > 0 ? { color: '#5cb85c' } : {}">
-            <strong>Progress: {{ (assessedSubmissions / submissions.length) * 100 | number: '0.0-0' }} % </strong>({{ assessedSubmissions }} out of
-            {{ submissions.length }} submitted models assessed)
-        </small>
+        <ng-container *ngIf="submissions && submissions.length === 0">
+            <small [ngClass]="'text-warning'">
+                {{ 'modelingAssessment.noSubmissions' | artemisTranslate: {} }}
+            </small>
+        </ng-container>
+        <ng-container *ngIf="submissions && submissions.length > 0 && exercise.assessmentType === AssessmentType.SEMI_AUTOMATIC">
```

Comment:
I am not that familiar with modeling exercises but is `AssessmentType.SEMI_AUTOMATIC` valid for this exercise type?

Reply:
I got confused with "Automatic assessment suggestions enabled". 
this is fixed in 55400db478c1f79a616da1f4ef1f342e471dad54